nowcasting_datamodel/read/read_gsp.py

```diff
@@ -19,6 +19,7 @@ def get_latest_gsp_yield(
     append_to_gsps: bool = False,
     regime: str = "in-day",
     datetime_utc: Optional[datetime] = None,
+    no_nans_in_capacities: bool = False,
```

Comment:
Can you change this to "allow_nans_in_capacities"> And reverse the logic? Otherwise I think it can be misintepreted as a double negative.

Reply:
Also is there a reason it should't be the default behavior to disallow NaNs?

Reply:
Yea, I will do that

Only trying to keep things the same, unless specifically changing something.

Reply:
I get the backward compatibility mindset, but is this the behaviour we want to conserve, or ever wanted in the first place..?

Reply:
I think it is behaviour we want. If pvlive suddenly starts putting nans as the capacity, then we dont want to use that, we want to use the most recent capacity that is not Nan

Note we still want the value from PVlive as we need a value every 30 mins to keep things going

Reply:
Ah, like we should avoid nans in the capacity. 

When we are just looking a yield values, we dont care if there are nans I'm the capacity or not